### PR TITLE
Add recipe for garlic toast

### PR DESCRIPTION
**Pull request by jjcrawford**
Added simple recipe for garlic toast. Currently no photos, can add some in about a week.

### CODE REVIEW

File: src/garlic-toast.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Garlic toast, perfect as a substitute for crackers in a cheese platter.
+
+- ⏲️ Prep time: 5 min
+- 🍳Cook time: 10-15 min
+
+## Ingredients
+
+- Sourdough loaf (sliced)
+- Garlic-infused olive oil
```
Comment (zyansheep) [inline src/garlic-toast.md:11]:
```suggestion
- Butter or Olive Oil (Optionally garlic-infused oil)
```
Butter also tastes good!

File: src/garlic-toast.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Garlic Toast
+
+Garlic toast, perfect as a substitute for crackers in a cheese platter.
+
+- ⏲️ Prep time: 5 min
+- 🍳Cook time: 10-15 min
+
+## Ingredients
+
+- Sourdough loaf (sliced)
```
Comment (zyansheep) [inline src/garlic-toast.md:10]:
```suggestion
- Any kind of firm, thin, sliced bread (Sourdough, Baguette)
```

File: src/garlic-toast.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+## Ingredients
+
+- Sourdough loaf (sliced)
+- Garlic-infused olive oil
+- Garlic salt
+
+## Directions
+
+1. Preheat oven to 180°C (356°F).
```
Comment (zyansheep) [inline src/garlic-toast.md:16]:
```suggestion
1. Preheat oven to 180°C (350°F).
```
Typical mapping from C to F in cooking is 180C = 350C

File: src/garlic-toast.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+2. Place sliced sourdough on a baking tray, prepared with aluminium foil
+3. Generously coat sourdough slices with garlic-infused olive oil.
+4. Season with garlic salt, to taste.
+5. Bake for 10-15 minutes at 180°C (356°F) or until crunchy throughout.
```
Comment (zyansheep) [inline src/garlic-toast.md:20]:
```suggestion
5. Bake for 10-15 minutes until crunchy throughout.
   - Alternatively set on broil for 1 minute.
```

File: src/garlic-toast.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+## Ingredients
+
+- Sourdough loaf (sliced)
+- Garlic-infused olive oil
+- Garlic salt
+
+## Directions
+
+1. Preheat oven to 180°C (356°F).
+2. Place sliced sourdough on a baking tray, prepared with aluminium foil
```
Comment (zyansheep) [inline src/garlic-toast.md:17]:
```suggestion
2. Place sliced bread on a baking tray, covered with aluminum foil

```

File: src/garlic-toast.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+3. Generously coat sourdough slices with garlic-infused olive oil.
```
Comment (zyansheep) [inline src/garlic-toast.md:18]:
```suggestion
3. Generously coat the bread slices with olive oil.
```

File: src/garlic-toast.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
+# Garlic Toast
+
+Garlic toast, perfect as a substitute for crackers in a cheese platter.
+
+- ⏲️ Prep time: 5 min
+- 🍳Cook time: 10-15 min
```
Comment (zyansheep) [inline src/garlic-toast.md:6]:
```suggestion
- 🍳Cook time: 10-15 min (or 1 minute on broil)
```

File: src/garlic-toast.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+2. Place sliced sourdough on a baking tray, prepared with aluminium foil
+3. Generously coat sourdough slices with garlic-infused olive oil.
+4. Season with garlic salt, to taste.
+5. Bake for 10-15 minutes at 180°C (356°F) or until crunchy throughout.
+6. Serve with cheese, charcuterie, pickles, dips, or to accommodate a soup. 
+
+## Contribution
+
+- Jack Crawford - [website](https://github.com/jjcrawford)
+
```
Comment (zyansheep) [inline src/garlic-toast.md:26]:
```suggestion
- Edited by Zyansheep

```

**Review by zyansheep — CHANGES_REQUESTED** [review]

**Comment by zyansheep** [comment]
This is what you are going for with garlic toast right? https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fwww.acouplecooks.com%2Fwp-content%2Fuploads%2F2020%2F03%2FGarlic-Toast-008.jpg&f=1&nofb=1

File: src/garlic-toast.md (hunk 5 above)
Comment (hallzy) [reply src/garlic-toast.md:17]: `aluminium` is actually the correct spelling for most of the world. I think Canada and the US may be the only places that don't. So I guess the question is, do we want to force American spelling.

File: src/garlic-toast.md (hunk 1 above)
Comment (jjcrawford) [reply src/garlic-toast.md:11]: you're probably not wrong, but it would hardly be garlic toast without the garlic-infused olive oil!

**Comment by jjcrawford** [comment]
> This is what you are going for with garlic toast right? https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fwww.acouplecooks.com%2Fwp-content%2Fuploads%2F2020%2F03%2FGarlic-Toast-008.jpg&f=1&nofb=1

basically, yeah

**Comment by jjcrawford** [comment]
@zyansheep I have no idea how broiling goes so I'm taking your word for it. Does it end up being crispy throughout? It's not very nice if it's only crispy on the outside and soft inside, that's basically just regular garlic bread. When done right, these things almost have the consistency of biscuits.

**Comment by zyansheep** [comment]
@jjcrawford It depends on the bread. I think I got confused when editing and forgot the difference between garlic toast and garlic bread. Broiling will crisp it on the outside and keep it soft on the inside (depending on the thickness & amount of oil) so its probably not desirable in most situations where you don't want a soft inside.

**Pull request merged**